packages/runtime/src/templates/getHandler.ts

```diff
@@ -140,6 +146,19 @@ const makeHandler = ({ conf, app, pageRoot, NextServer, staticManifest = [], mod
       event.headers['accept-language'] = event.headers['accept-language'].replace(/\s*,.*$/, '')
     }
 
+    if (context?.clientContext?.custom?.blobs) {
+      const rawData = Buffer.from(context.clientContext.custom.blobs, 'base64')
+      const data = JSON.parse(rawData.toString('ascii'))
+      setBlobInit({
+        authentication: {
+          contextURL: data.url,
+          token: data.token,
+        },
+        context: `deploy:${event.headers['x-nf-deploy-id']}`,
+        siteID: event.headers['x-nf-site-id'],
+      })
```

Comment:
We use Blob Storage in a place that doesn't have direct access to `event`/`context` which is needed to initialize the store, so instead we globally set init argument on each request. This means it's possible that requests would use init data we got from another one (depending on timing), but this is acceptable/ok to do and only note is that there is expiration time on token so that's why this being refreshed on each request and not just once.